### PR TITLE
Disable shadows when taking goldens

### PR DESCRIPTION
**Pull request by Hixie**
New goldens: https://github.com/flutter/goldens/commit/a3923c1a6ed192ad55f18bef6999422d11fd7b1b

### CODE REVIEW

**Comment by Hixie** [comment]
Fixes https://github.com/flutter/flutter/issues/17262

**Comment by tvolkert** [comment]
Bots aren't happy - among the failures is "MergeableMaterial paints shadows", so it seems like some tests need updating.

File: packages/flutter/lib/src/rendering/proxy_box.dart [hunk 1]
```diff
@@ -1591,20 +1591,34 @@ class RenderPhysicalModel extends _RenderPhysicalModelBase<RRect> {
...
+      final Path offsetRRectAsPath = new Path()..addRRect(offsetRRect);
+      bool paintShadows = true;
+      assert(() {
+        if (debugDisableShadows) {
+          context.canvas.drawRRect(
+            offsetRRect,
+            new Paint()
+              ..color = shadowColor
+              ..style = PaintingStyle.stroke
+              ..strokeWidth = elevation * 2.0,
```
Comment (tvolkert) [inline packages/flutter/lib/src/rendering/proxy_box.dart]: Why * 2?

**Review by tvolkert — APPROVED** [review]
LGTM!

(modulo existing tests that need updating)

File: packages/flutter/lib/src/rendering/proxy_box.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter/lib/src/rendering/proxy_box.dart]: To make the difference more obvious. It was just a guess as to what would look good, I'm happy to change it to 1.0 if you like.

Comment (tvolkert) [reply packages/flutter/lib/src/rendering/proxy_box.dart]: That's what I was figuring.  2.0 sgtm

**Comment by tvolkert** [comment]
LGTM

**Review by tvolkert — APPROVED** [review]

**Pull request merged**